docs/guides/Redux-Integration.md

```diff
@@ -85,3 +85,35 @@ To make jest tests work with your react-navigation app, you need to change the j
   ]
 }
 ```
+
+## Extra : Handling back button in android
```

Comment:
Can you drop the "Extra" from the section title? Can you also change "Handling back button in android" to "Handling the Hardware Back Button in Android"? This will make it less ambiguous (i.e. does it refer to react-navigation's Back button or the Android hardware back button?).

Reply:
Was android hardware